src/bull.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// eslint-disable-next-line no-console
+const { randomNumber } = require('./getRandom');
+
+function bull(param) {
+  let count = 0;
+  const arrUser = param.split('');
+  const arrPC = randomNumber.split('');
+
+  arrUser.filter((item, index) => {
```

Comment:
What exactly are you filtering if you return `undefined` for each element of the array. Use `forEach` or loops